src/Stratis.Features.FederatedPeg/Controllers/CollateralVotingController.cs

```diff
@@ -40,6 +40,9 @@ public CollateralVotingController(IFederationManager fedManager, ILoggerFactory
         [ProducesResponseType((int)HttpStatusCode.InternalServerError)]
         public IActionResult VoteAddFedMember([FromBody]CollateralFederationMemberModel request)
         {
+            if (this.network.Consensus.ConsensusFactory is CollateralPoAConsensusFactory)
```

Comment:
It looks like we are only ever using `SmartContractCollateralPoAConsensusFactory` which inherits from `CollateralPoAConsensusFactory` so is this needed?

Reply:
It can be removed I think.